ios/Share Extension/ShareViewController.swift

```diff
@@ -0,0 +1,360 @@
+//
```

Comment:
TODO: This file still needs to be cleaned up as it is retrieved from the template of the package.